opta/commands/config.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional
+
+import click
+from click_didyoumean import DYMGroup
+
+from opta.core.aws import AWS
+from opta.core.azure import Azure
+from opta.core.cloud_client import CloudClient
+from opta.core.gcp import GCP
+from opta.utils import logger
+
+
+@click.group(cls=DYMGroup)
+def config() -> None:
+    pass
+
+
+cloud_option = click.option(
+    "-c",
```

Comment:
let's not use `-c` because we use it everywhere else for `--config` but this has a different meaning here
it's fine to only support `--cloud` not shortcut